zipkin-ui/js/component_data/dependency.js

```diff
@@ -45,37 +45,37 @@ export default component(function dependency() {
     });
   };
 
-  this.after('initialize', function () {
-    this.on(document, 'dependencyDataRequested', function (event, {endTs, lookback}) {
+  this.after('initialize', function() {
+    this.on(document, 'dependencyDataRequested', function(event, {endTs, lookback}) {
       this.getDependency(endTs, lookback);
     });
 
-    this.on(document, 'serviceDataRequested', function (event, {serviceName}) {
-      this.getServiceData(serviceName, function (data) {
+    this.on(document, 'serviceDataRequested', function(event, {serviceName}) {
+      this.getServiceData(serviceName, data => {
         this.trigger(document, 'serviceDataReceived', data);
-      }.bind(this));
+      });
     });
 
-    this.on(document, 'parentChildDataRequested', function (event, {parent, child}) {
-      this.getDependencyData(parent, child, function (data) {
+    this.on(document, 'parentChildDataRequested', function(event, {parent, child}) {
+      this.getDependencyData(parent, child, data => {
         this.trigger(document, 'parentChildDataReceived', data);
-      }.bind(this));
```

Comment:
One of the arrow syntax perks: We don't have to worry about binding "this" all the time :smile: